NewAndNoteworthy/CHANGELOG-API.md

```diff
@@ -665,3 +665,11 @@ The class BuiltinDetctionArgsGeneric will be removed. Use the correctly
 spelled BuiltinDetectionArgsGeneric instead.
 
 - org.eclipse.cdt.jsoncdb.core.participant.Arglets.BuiltinDetctionArgsGeneric
+
+## API Removals after June 2025
```

Comment:
I have simplified the API document by removing requirement to double-enter the contents - see #359